### ISSUE TITLE
Pseudoterminal job control

### ISSUE THREAD

**Issue by amscanne** [google/gvisor#154]
gVisor PTYs must implement job control and related features. These include sessions, foreground/background processes, and signal handling.

**Comment by nhooyr** [comment]
Is this issue related?

```
docker run --runtime=runsc -it --rm python python -c 'import pty; pty.spawn("/bin/sh")'
```

shows
```
/bin/sh: 0: can't access tty; job control turned off
```

**Comment by kevinGC** [comment]
At this point there is basic job control support implemented by #585 and #803. Closing this in favor of bugs like #422 that track specific features.

#6148 has been created to track the specific TODO still referring to this bug.
